modules/node_modules/@ncigdc/components/modals/SelectModal.js

```diff
@@ -0,0 +1,95 @@
+// @flow
+
+import React from 'react';
+import { compose, withState } from 'recompose';
+
+import { Row, Column } from '@ncigdc/uikit/Flex';
+import Button from '@ncigdc/uikit/Button';
+import { visualizingButton } from '@ncigdc/theme/mixins';
+
+const styles = {
+  horizonalPadding: {
+    paddingRight: 20,
+    paddingLeft: 20,
+  },
+  button: {
+    ...visualizingButton,
+    minWidth: 100,
+  },
+};
+
+type TOption = {
+  name: string,
+};
+
+type TProps = {
+  options: Array<TOption>,
+  onClose: Function,
+  selectedOptions: Array<number>,
+  setSelectedOptions: Function,
+};
+
+const SelectOverlay = ({ options = [], onClose, selectedOptions, setSelectedOptions }: TProps) => (
+  <Column>
+    <Row
+      style={{
+        ...styles.horizonalPadding,
+        borderBottom: '1px solid #e5e5e5',
+      }}
+    >
+      <h2 style={{ marginTop: 0 }}>Select tracks to add</h2>
+    </Row>
+    <div
+      style={{
+        ...styles.horizonalPadding,
+        paddingTop: 10,
+      }}
+    >
+      {
+        options.map((track, i) => (
+          <Row key={track.name}>
+            <input
+              value={i}
+              type="checkbox"
+              name={track.name}
+              id={track.name}
+              checked={selectedOptions.indexOf(i) >= 0}
+              onChange={() => {
+                const index = selectedOptions.indexOf(i);
```

Comment:
I can't seem to uncheck a track (clicking once selects a track, clicking twice does not de-select it)

I wonder if it's due to not checking for the value here and triggering on "onChange"